reference/promotions.v3.yml

```diff
@@ -1398,6 +1494,13 @@ components:
                     current_uses: 0
                     created: '2019-01-20T22:00:00+00:00'
                 meta:
+                  cursor_pagination:
+                    count: 1
+                    per_page: 50
+                    start_cursor: 'eyJpZCI6IjIzNzU1NyJ9'
+                    end_cursor: 'eyJpZCI6IjIzNzU1MyJ9'
+                    links:
+                      next: '?limit=5&after=eyJpZCI6IjIzNzU1NyJ9'
```

Comment:
if there is only 1 item then this is the last page, and there should not be a next page right 😄

Reply:
For this endpoint the logic is more 'naive' then that 😅 So no we will still provide a "next" page even if the result set is smaller than the limit.